.changeset/rude-geese-carry.md

```diff
@@ -0,0 +1,5 @@
+---
+'@recative/capacitor-geckoview': major
+---
+
+replace android system webview via geckview
```

Comment:
1. Please follow the rule of Conventional Commits
2. Incorrect grammar: https://tw.dictionary.search.yahoo.com/search?p=replace+with
3. Your changelog does not correctly reflect the changes corresponding to this commit. The problem that your commit focuses on is: "Fix the problem that Ionic and GeckoView cannot communicate with each other"